### PR TITLE
Added script to export XML with all information

### PR DESCRIPTION
**Pull request by pfedotovsky**
Added script to export XML will all information

### CODE REVIEW

File: src/XmlUtility.ps1 [hunk 1]
```diff
@@ -0,0 +1,128 @@
+﻿$WikiConfig = @{
```
Comment (AnatolyKulakov) [inline src/XmlUtility.ps1]: You should't use the `Wiki` keyword here. This file is not associated to a Wiki.

File: src/XmlUtility.ps1 [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+$WikiRepository = @{
+    Communities = @{}
+    Meetups = @{}
+    Talks = @{}
+    Speakers = @{}
+    Friends = @{}
+    Venues = @{}
+}
+
+function Test-WikiEnvironment()
```
Comment (AnatolyKulakov) [inline src/XmlUtility.ps1]: You don't use artifacts that created this method. Do you really need it?

File: src/XmlUtility.ps1 [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    WikiDir = Resolve-FullPath $Config.RootDir '..\..\SpbDotNet.wiki'
+}
+
+$WikiRepository = @{
+    Communities = @{}
+    Meetups = @{}
+    Talks = @{}
+    Speakers = @{}
+    Friends = @{}
+    Venues = @{}
```
Comment (AnatolyKulakov) [inline src/XmlUtility.ps1]: You don't need a dictionaries. You have enough lists.

File: src/XmlUtility.ps1 [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    $xRoot.Add((Convert-ToXml -Entities $WikiRepository.Meetups.Values -EntitiesName "Meetups"))
+    $xRoot.Add((Convert-ToXml -Entities $WikiRepository.Talks.Values -EntitiesName "Talks"))
+    $xRoot.Add((Convert-ToXml -Entities $WikiRepository.Speakers.Values -EntitiesName "Speakers"))
+    $xRoot.Add((Convert-ToXml -Entities $WikiRepository.Friends.Values -EntitiesName "Friends"))
+    $xRoot.Add((Convert-ToXml -Entities $WikiRepository.Venues.Values -EntitiesName "Venues"))
+
+    $xdoc = [System.Xml.Linq.XDocument]::new()
+    $xDoc.Add($xRoot)
+
+    $xDoc.Save($Config.ArtifactsDir + "/common.xml")
```
Comment (AnatolyKulakov) [inline src/XmlUtility.ps1]: You can use [Join-Path](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/join-path?view=powershell-5.1) cmdlet for cross-platform path concatenation.

**Comment by pfedotovsky** [comment]
Thanks for the comments! Have you taken a look at the file ending issue (AppVeyor build is failing)?

**Comment by AnatolyKulakov** [comment]
Nope, I have plan to do it on the weekend

**Comment by AnatolyKulakov** [comment]
Hey @pfedotovsky . I investigate the failed test ("Should have windows like end of file"). And found that your file "XmlUtility.ps1" was actually pushed in the unix end of line format.  It's easy to see, just download raw file (do not use git client) from this link:

https://raw.githubusercontent.com/pfedotovsky/Boombr/d63912c5ac3d92ac9c2524a1c2b7296e7720621c/src/XmlUtility.ps1

and open it in any hex editor. You will see 0x0A line ending.

You can't detect this problem through the git client because it automatically convert the end of lines.
For more detail see `autocrlf` option (`git config --help`).
For having the same line ending in your local copy and in the remote repository, you should have value `autocrlf = false`.

**Comment by AnatolyKulakov** [comment]
@pfedotovsky , any updates?

**Comment by pfedotovsky** [comment]
@AnatolyKulakov planning to fix all of this soon (today or tomorrow)

File: src/XmlUtility.ps1 (hunk 1 above)
Comment (pfedotovsky) [reply src/XmlUtility.ps1]: Fixed

File: src/XmlUtility.ps1 (hunk 3 above)
Comment (pfedotovsky) [reply src/XmlUtility.ps1]: Fixed

File: src/XmlUtility.ps1 (hunk 2 above)
Comment (pfedotovsky) [reply src/XmlUtility.ps1]: No, I've removed it

File: src/XmlUtility.ps1 (hunk 4 above)
Comment (pfedotovsky) [reply src/XmlUtility.ps1]: Fixed

**Pull request merged**